auth/src/main/res/values/config.xml

```diff
@@ -9,6 +9,16 @@
     -->
     <string name="facebook_application_id" translatable="false">CHANGE-ME</string>
 
+    <!--
+     To enable Chrome Custom Tabs for Facebook Login on devices without the Facebook app
```

Comment:
In my testing when you have a misconfigured custom tabs setup (scheme = `fbYOUR_APP_ID`) it acts as if you had no custom tabs setup and falls back onto the normal webview flow.

Likely the FB SDK asks the system to handle the URL with your App ID, the system says there are no registered intent handlers, and it falls back.